SLP clarification (#269)
vippsas/vipps-developers: common-topics/vipps-landing-page.md

Vipps. The `fallback` URL sent in the API request can therefore be the
merchant's main URL, like `https://example.com`, etc.

**Important:** When using `"skipLandingPage": true` in the API request that
initiates the payment:
* Vipps will send a push notification immediately to the Vipps app
  for the user with the specified phone number, without showing the landing page.
* It is crucial to use the correct format for the user's phone number.
  If not, the payment will fail.
* The user is not able to provide a different phone number for completing the

>   payment. This means that a "Vipps for those under 15" (that can not pay

Reviewer: ```suggestion
  payment. This means that a "Vipps for those under 15" (that cannot pay
```

>   businesses) can not have someone else pay for them.

Reviewer: ```suggestion
  businesses) cannot have someone else pay for them.
```

> * The user is not sent to a `fallback` URL (the result page) after completion

Reviewer: Is that supposed to say "now"? Because, it seems like we are talking about what happens on the fallback page below

Author: Fixed it up bit, thanks.

Reviewer: You have a typo below :-p  "Instead of the he "result page""
but it is more understandable

Reviewer: I will fix it so we don't have to think about it

  of the payment.
  The "result page" is just the confirmation in the Vipps app.
  The `fallback` URL sent in the API request can simply be the website URL.
* If the sale unit is not whitelisted, the request will fail and an error
  message will be returned.

## Sequence diagram

This sequence diagram shows the difference between the normal flow and
the flow with `"skipLandingPage": true`:

```mermaid
sequenceDiagram
    participant M as Merchant
    participant API as Vipps API
    participant LP as Landing page
    participant App as Vipps app
    actor U as User
    M-->>API: Initiate payment<br/>with or without skipLandingPage
    alt Normal flow
        API-->>LP: Show the landing page

>         U-->>LP: Confirm/enter/change phone number

Author: We're too old, @rebekaburnett! And even I don't specify mobile/landline anymore.

>         U-->>LP: Confirm/enter/change phone number

Reviewer: Oh, that I even thought of that means I'm over-the-hill. haha
